#Request parameters
float64 x
float64 y
float64 theta
--------------------------------
#CREATE response parameters
bool status